definitions/linux.yaml

```diff
@@ -450,3 +450,12 @@ urls:
   - 'https://www.kernel.org/doc/Documentation/filesystems/sysfs-pci.txt'
   - 'https://wiki.debian.org/HowToIdentifyADevice/PCI'
 supported_os: [Linux]
+---
+name: ResolvConfFile
```

Comment:
Lets stick DNS in the name to make it easier to discover:

DNSResolvConfFile
doc: DNS resolver configuration file

Also move it up so it's in alphabetical order with the other artifacts.